docs/participant_information.md

```diff
@@ -4,70 +4,67 @@ nav_title: Logistics
 ---
 
 
-**Coming soon:** Find helpful information related to health and safety and the workshop location.
-
+Find helpful information about the workshop location, travel reimbursement, and healthy and safety.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-<!--
-
 ### Table of Contents
 
-- [COVID-19 Safety Policy](#covid-19-safety-policy)
+- [Action Items](#action-items)
 - [Workshop Location](#workshop-location)
-  - [Parking](#parking)
-  - [Nearby Lodging](#nearby-lodging)
 - [Travel Reimbursement](#travel-reimbursement)
-
--->
+- [Health and Safety](#health-and-safety)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-<!--
+## Action Items
 
-## COVID-19 Safety Policy
+* Please review and sign the [training waiver](https://app.hellosign.com/s/2o2yghCj) before the workshop begins.
+* [Visit the meal form](https://forms.gle/aTfFxAo8R3zucsMs6) to tell us about any dietary restrictions and to RSVP for the group dinner. 
+* [Join the Slack group](http://ccdatalab.org/slack) that we will use for communication purposes and to stay in touch after the workshop. We will add you to the `#2023-october-training` channel.
 
-Please be aware that with the goal of keeping everyone safe, healthy, and comfortable, we will comply with ALSF’s COVID-19 safety policy during this workshop. 
-The following protocols and practices will be enforced: 
+## Workshop Location
 
-* You will be required to wear a surgical or respirator mask in the meeting room used for training. 
-We will have extra surgical masks available during the workshop should you forget to wear one or wish to change your mask during the day.
-* Hand sanitizer will be made available and HEPA filters will be used in the meeting room used for training.
-* Weather permitting, we will aim to hold our lunch break outside. 
-Outdoor tables and seating are available at the workshop location.
-* If you are not feeling well, please do not attend the workshop. 
-We will not withhold refunds if a participant must cancel because they are not feeling well. 
+**The workshop will be held at The Inn at Penn, a Hilton Hotel, located at [3600 Sansom St, Philadelphia, PA 19104](https://www.google.com/maps/place/The+Inn+at+Penn,+a+Hilton+Hotel/@39.9537829,-75.1983565,17z/data=!3m1!4b1!4m9!3m8!1s0x89c6c6575236ba05:0x2ab1dcf9ee499b6b!5m2!4m1!1i2!8m2!3d39.9537829!4d-75.1957816!16s%2Fg%2F1hc24w67l?entry=ttu)**. 
+We will meet in Conference Room 603. 
+If you have trouble finding the building or the room, please use Slack to direct message `Jen O'Malley`. 
 
-## Workshop Location
+The Inn at Penn is located in West Philadelphia in the [University City](https://www.universitycity.org/) area.
+The neighborhood is well-known for being home to the University of Pennsylvania and Drexel University.
+It is a largely walkable area with plenty of restaurants and attractions! 
```

Comment:
please consider this comment my vote for [Manakeesh](https://www.manakeeshcafe.com/) 🤤

Reply:
Wait they do catering?! https://order.toasttab.com/online/manakeesh-cafe-bakery-grill#dfca412c4-d782-4a16-9562-1531165a4f18d63ce381c-e43e-4ff9-8b26-85d64733b21b
please therefore consider this my vote to get them to cater something at some point in the future ;)